dp-cluster-setup-utility.py

```diff
@@ -1062,7 +1210,7 @@ def current_user(self):
 
   knox = Knox(user.url_input('Knox URL that is network accessible from DataPlane', 'knox.url', default=str(ambari.cluster.knox_url())), knox_user=ambari.cluster.knox_user(), knox_group=ambari.cluster.knox_group())
 
-  topologies_to_deploy = [TokenTopology(dp.public_key()), DpProxyTopology(ambari, dp.dependency_names())]
+  topologies_to_deploy = [TokenTopology(dp.public_key()), DpProxyTopology(ambari, dp.dependency_names()), BeaconProxyTopology(ambari, dp.dependency_names())]
```

Comment:
We should be deploying BeaconProxyTopology only if DLM is being installed.

Reply:
Will fix this.